### ISSUE TITLE
Documentation: App level alert> bg color of warning alert is wrong

### ISSUE THREAD

**Issue by lil-kim** [vmware/clarity#3604]
## App level alert - bg color of warning alert is wrong.

The BG color used to be a rust color which was chosen because it met contrast requirements with white text. It's now a much lighter color with white text and fails contrast a11y. It should be reverted back to the rust color: #C25400

1.  Go to https://clarity.design/documentation/alerts
![image.png](https://images.zenhubusercontent.com/5840974cb9dc3c622022d1df/374a4338-c95f-43e3-9741-66dbaaa3b6de)

## Expected behavior

Revert the color back to:
#C25400
It should look like this:
![image.png](https://images.zenhubusercontent.com/5840974cb9dc3c622022d1df/e8a9095a-4c89-46ab-b322-349dfd4d9ef3)

**Comment by mathisscott** [comment]
I've verified this is not an issue in 1.x.
